### PR TITLE
Add support for exempted identities in non-PUIS Asset transfer restrictions

### PR DESCRIPTION
**Pull request by monitz87**

### CODE REVIEW

File: src/api/entities/Identity/__tests__/index.ts [hunk 1]
```diff
@@ -704,6 +705,7 @@ describe('Identity class', () => {
     let rawScopeId: ScopeId;
 
     let stringToTickerStub: sinon.SinonStub<[string, Context], Ticker>;
+    let scopeIdOfStub: sinon.SinonStub;
```
Comment (VictorVicente) [inline src/api/entities/Identity/__tests__/index.ts:708]: There is a warning here as 'scopeIdOfStub' is defined but never used

File: src/types/index.ts [hunk 2]
```diff
@@ -1216,16 +1216,16 @@ export interface ProcedureAuthorizationStatus {
...
   /**
-   * array of Scope IDs that are exempted from the Restriction
+   * array of Scope/Identity IDs that are exempted from the Restriction
+   *
+   * @note if the Asset requires investor uniqueness, Scope IDs are used. Otherwise, we use Identity IDs. More on Scope IDs and investor uniqueness
+   *   {@link https://developers.polymesh.network/introduction/identity#polymesh-unique-identity-system-puis | here} and
+   *   {@link https://developers.polymesh.network/polymesh-docs/primitives/confidential-identity | here}
    */
-  exemptedScopeIds?: string[];
+  exemptedIds?: string[];
```
Comment (VictorVicente) [inline src/types/index.ts:1225]: Is it ok we are not supporting `Identity[]` here?

Comment (monitz87) [reply src/types/index.ts:1225]: The value could be a scopeId. Even if we check whether the asset has PUIS enabled or not, the agent could have exempted a ScopeId, invalid DID, or anything that fits the format

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**